### DISCUSSION TITLE
Ultimaker Cura 5.2.1 and MP10

### DISCUSSION THREAD

**Discussion by Bidouyiou** [community/community#40769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hi everyone, could someone tell me why 3D printer Monoprice MP10 is no more supported in Ultimaker Cura 5.2.1 ?
Thanks in advance for your reply

![Cura 5 2 1](https://user-images.githubusercontent.com/119807121/205505489-1e544071-1f8d-4dfa-a07c-eb8af9830155.jpg)

**Comment by mohamedzhioua** [comment]
here  a presentation of Dr. Vax where explain step and step on setting
 https://docs.google.com/presentation/d/10Two5vMClDXDwZXuJUp7YF1bjKibKQVjw1eilUuuLaY/edit#slide=id.gc6f9e470d_0_0
